packages/flutter/test/material/list_tile_test.dart

```diff
@@ -499,6 +511,15 @@ void main() {
     // A selected ListTile's InkWell gets the ListTileTheme's shape
     await tester.pumpWidget(buildFrame(selected: true, shape: roundedShape));
     expect(inkWellBorder(), roundedShape);
+
+    // Cursor updates when hovering disabled ListTile
```

Comment:
Shouldn't we `buildFrame(enabled: false)` first?

Reply:
It was already `enabled: false` in https://github.com/flutter/flutter/blob/03bc96deb2a3fd1e2ae5c300011ec93c2fc97e0f/packages/flutter/test/material/list_tile_test.dart#L492 so I missed it, thanks.